build/targets/KoreBuild.DefaultBuildSettings.props

```diff
@@ -1,4 +1,16 @@
 <Project>
+  <!-- pickup build number from CI if available -->
+  <PropertyGroup Condition=" '$(BuildNumber)' == '' ">
+    <!--
+      TeamCity=https://confluence.jetbrains.com/display/TCD10/Predefined+Build+Parameters
+      AppVeyor=https://www.appveyor.com/docs/environment-variables/
+      Travis=https://docs.travis-ci.com/user/environment-variables/
+    -->
+    <BuildNumber Condition=" '$(BUILD_NUMBER)' != '' ">$(BUILD_NUMBER)</BuildNumber>
```

Comment:
If we have this in any of our build chains:

1. A => `Build_Number` => `build.counter`
2. B => `Build_Number` => `A.Build_Number`

Removing `Build_Number` would no longer be correct. Just wanted you to ensure we weren't doing this.